cmd/jaeger/internal/integration/kafka_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/jaegertracing/jaeger/plugin/storage/integration"
+)
+
+func TestKafkaStorage(t *testing.T) {
+    integration.SkipUnlessEnv(t, "kafka")
+
+    collectorConfig := "../../collector-with-kafka.yaml"
+    ingesterConfig := "../../ingester-remote-storage.yaml"
+
+    collector := &E2EStorageIntegration{
+        SkipStorageCleaner: true,
+        ConfigFile: collectorConfig,
+        StorageIntegration: integration.StorageIntegration{
```

Comment:
Since you are running tests on the ingester object, these settings are irrelevant for collector